python/cudf/cudf/core/series.py

```diff
@@ -1568,6 +1568,8 @@ def _normalize_binop_value(self, other):
         elif isinstance(other, Index):
             return Series(other)._column
         else:
+            if isinstance(other, np.ndarray) and other.ndim == 0:
+                other = other.item()
```

Comment:
Keep moving it down 😄. Move it into `Column.normalize_binop_value`.

Reply:
Main class Column doesn't have `normalize_binop_value`, might have to add in Main class and the call children api
.

Reply:
Generally we should clean up `normalize_binop_value` to deduplicate all of the different Column classes, but lets do that in a follow up not right before release. For now, I guess we can duplicate the necessary logic into the different Column classes.